src/pt/exhentai/src/eu/kanade/tachiyomi/extension/pt/exhentai/ExHentai.kt

```diff
@@ -0,0 +1,174 @@
+package eu.kanade.tachiyomi.extension.pt.exhentai
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.network.asObservableSuccess
+import eu.kanade.tachiyomi.network.interceptor.rateLimit
+import eu.kanade.tachiyomi.source.model.Filter
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.ParsedHttpSource
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import okhttp3.Response
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import rx.Observable
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class ExHentai : ParsedHttpSource() {
+
+    override val name = "ExHentai"
```

Comment:
rename source & extension to "ExHentai.net.br" or similar to avoide confusion with exh/sadpanda